src/main/java/org/everit/jira/timetracker/plugin/JiraTimetrackerChartWebAction.java

```diff
@@ -353,7 +354,12 @@ private boolean loadDataFromSession() {
     if (data == null) {
       return false;
     } else {
-      ReportSessionData reportSessionData = (ReportSessionData) data;
+      ReportSessionData reportSessionData = null;
+      try {
+        reportSessionData = (ReportSessionData) data;
+      } catch (ClassCastException e) {
```

Comment:
Please use instanceof rather than communicating with exception and than swallowing it.